test/tests/integration.test.ts

```diff
@@ -14,39 +14,35 @@
  * limitations under the License.
  */
 
+import { cp } from "fs/promises";
+import assert from "node:assert";
 import { readFile, rm, writeFile } from "node:fs/promises";
 import { join } from "node:path";
 
-import { FS_OPTIONS, PACKAGE_JSON_FILE_NAME } from "../src/lib/const.js";
-
+import { FS_OPTIONS, PACKAGE_JSON_FILE_NAME } from "../../src/lib/const.js";
+import { fluenceEnv, NO_PROJECT_TEST_NAME } from "../helpers/constants.js";
 import {
-  NO_PROJECT_TEST_NAME,
-  fluence,
-  fluenceEnv,
-  getMultiaddrs,
-  init,
-  maybeConcurrentTest,
-} from "./helpers.js";
-
-const multiaddrs = await getMultiaddrs();
+  initializeTemplate,
+  multiaddrs,
+  runAquaFunction,
+} from "../helpers/sharedSteps.js";
 
 describe("integration tests", () => {
-  maybeConcurrentTest("should work with minimal template", async () => {
+  test.concurrent("should work with minimal template", async () => {
```

Comment:
the benifit of using maybeConcurentTest here is that you can turn it off with an env variable if you want to run tests sequentially to see the logs clearly. But I understand that here it's impossible to turn off easilly now. I am I correct? If so, does `maybeConcurentTest` cause any problems or something? Why change that?

Reply:
@shamsartem i proposed to do so [here](https://github.com/fluencelabs/cli/pull/731#discussion_r1462162235).
Seems like the test execution is still configurable via env var. Please look at 2 previous Dmitriy's commits - he managed to configure test concurrency via env. @vasd85 can you confirm that?

Reply:
hm so it's just controled with jest config `maxWorkers`. If this works - very nice

Reply:
Yes, the env var still works. I moved this "switcher" in the `jest.config.js` - `maxWorkers: process.env[RUN_TESTS_IN_PARALLEL] === "false" ? 1 : 5,`. If we run with 1 worker there is no way to parallelize tests.